spec/controllers/api/schedules/schedules_destroy_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Api::SchedulesController do
+  include Devise::TestHelpers
+
+  describe '#destroy' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'deletes a schedule' do
+      sign_in user
+      schedule = FactoryGirl.create(:schedule, user: user)
+      before = Schedule.count
+      delete :destroy, id: schedule._id.to_s
+
+      expect(response.status).to eq(200)
+      expect(before > Schedule.count).to be_truthy
+    end
+
+    it 'prevents unauthorized deletion' do
+      sign_in user
+      schedule = FactoryGirl.create(:schedule)
+      delete :destroy, id: schedule._id.to_s
+      before = Schedule.count
+
+      expect(response.status).to eq(403)
+      expect(before == Schedule.count).to be_truthy
+      expect(json[:error]).to include("Not your schedule.")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.